src/renderer/PovSelection.tsx

```diff
@@ -125,26 +104,19 @@ export default function PovSelection(props: IProps) {
      */
     const getCloudIcon = () => {
       let opacity = 1;
-      let title = 'Saved in the cloud';
-
-      let onClick = (
-        event: React.MouseEvent<HTMLButtonElement, MouseEvent>
-      ) => {
-        handleChangePov(event, cloudIndex);
-      };
+      let title = 'Use cloud version';
 
       if (!haveCloudVideo) {
         opacity = 0.2;
         title = 'No cloud recording is saved';
```

Comment:
Mousing over a disabled icon doesn't show this tooltip anymore.

Reply:
Made them show with some tailwind magic.